lib/extensions/sunspot/lib/sunspot/dsl/field_query.rb

```diff
@@ -0,0 +1,12 @@
+module Sunspot
```

Comment:
Where did all these extensions come from? If they were created for the KPI work, they will need tests. If they were ported from an external gem, I'd prefer to use the external gem